scripts/Dockerfile.alpine.build

```diff
@@ -30,9 +30,16 @@ WORKDIR /tmp/dd/datadog-agent/"${CMD_PATH}"
 
 RUN --mount=type=cache,target=/go/pkg/mod \
     --mount=type=cache,target=/root/.cache/go-build \
+    if [ -z "$AGENT_VERSION" ]; then \
     /usr/lib/go/bin/go build -ldflags="-w \
     -X github.com/DataDog/datadog-agent/pkg/serverless/tags.currentExtensionVersion=$EXTENSION_VERSION" \
-    -tags "${BUILD_TAGS}" -o datadog-agent;
+    -tags "${BUILD_TAGS}" -o datadog-agent; \
+    else \
+    /usr/lib/go/bin/go build -ldflags="-w \
+    -X github.com/DataDog/datadog-agent/pkg/serverless/tags.currentExtensionVersion=$EXTENSION_VERSION \
+    -X github.com/DataDog/datadog-agent/pkg/version.agentVersionDefault=$AGENT_VERSION" \
+    -tags "${BUILD_TAGS}" -o datadog-agent; \
+    fi
```

Comment:
nit.

Can we indent some of these lines. Real hard to read otherwise 😄

Reply:
Yes, I'll make sure to indent in `Dockerfile.build` as well then